### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by keileg**
## Proposed changes
Added a file with guidelines for how to contribute.

Also removed old files related to pypi, and deleted the file Papers.md (which had not been updated in three years).

## Types of changes

What types of changes does this PR introduce to PorePy?
_Put an `x` in the boxes that apply._

- [ ] Minor change (e.g., dependency bumps, broken links).
- [ ] Bugfix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Testing (contribution related to testing of existing or new functionality).
- [ ] Documentation (contribution related to adding, improving, or fixing documentation).
- [ ] Maintenance (e.g., improve logic and performance, remove obsolete code).
- [x] Other: GH guidelines.

## Checklist

_Put an `x` in the boxes that apply or explain briefly why the box is not relevant._
None are relevant.

- [ ] The documentation is up-to-date.
- [ ] Static typing is included in the update.
- [ ] This PR does not duplicate existing functionality.
- [ ] The update is covered by the test suite (including tests added in the PR).
- [ ] If new skipped tests have been introduced in this PR, `pytest` was run with the `--run-skipped` flag.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Contributions to `PorePy` are always welcome, in the form of [pull requests](https://github.com/pmgbergen/porepy/pulls), bug reports or other [issues](https://github.com/pmgbergen/porepy/issues/new), or [questions](https://github.com/pmgbergen/porepy/discussions/new/choose). 
+
+When making a **pull request** we ask you to verify that the code follow the existing practices in `PorePy`:
```
Comment (IvarStefansson) [inline CONTRIBUTING.md:3]:
```suggestion
When making a **pull request** we ask you to verify that the code follows the existing practices in `PorePy`:
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+ * All new functionaity should be covered by tests, and the tests placed in the appropriate location: The tests associated with a file src/porepy/path/to/file.py should be located in tests/path/to/test_file.py. If you are unsure of what amounts to proper test coverage, please get in touch.
+ * All functions should be documented following the [guidelines](https://pmgbergen.github.io/porepy/html/docsrc/howto/howto-docstring.html).
+ * All code should provide type hints, and be formatted following black, flake8 and isort.
```
Comment (IvarStefansson) [inline CONTRIBUTING.md:6]:
```suggestion
 * All code should provide type hints and be formatted following black, flake8 and isort.
```

**Review by IvarStefansson — APPROVED** [review]
Nice.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+Contributions to `PorePy` are always welcome, in the form of [pull requests](https://github.com/pmgbergen/porepy/pulls), bug reports or other [issues](https://github.com/pmgbergen/porepy/issues/new), or [questions](https://github.com/pmgbergen/porepy/discussions/new/choose). 
+
+When making a **pull request** we ask you to verify that the code follows the existing practices in `PorePy`:
+ * All new functionaity should be covered by tests, and the tests placed in the appropriate location: The tests associated with a file src/porepy/path/to/file.py should be located in tests/path/to/test_file.py. If you are unsure of what amounts to proper test coverage, please get in touch.
+ * All functions should be documented following the [guidelines](https://pmgbergen.github.io/porepy/html/docsrc/howto/howto-docstring.html).
+ * All code should provide type hints and be formatted following black, flake8 and isort.
```
Comment (OmarDuran) [inline CONTRIBUTING.md:6]:
```suggestion
 * All code should provide type hints statically checked with Mypy and be formatted following black, flake8, and isort.
```

**Review by OmarDuran — APPROVED** [review]
Nice work. I placed two minor suggestions.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+All pull requests will undergo code review, and will also be tested against the existing test suite and checked for style consistency.
+By submitting a pull request, you are licensing the code under `PorePy`'s [license](https://github.com/pmgbergen/porepy/blob/develop/LICENSE), and affirm that you own the copyright of the code or have permission to distribute the code under the licence. 
+
+If reporting an **issue**, we ask that you provide a minimal working example to reproduce the bug, together with other information that could be useful for debugging.
```
[the remark this answers is not in the document]
Comment (IvarStefansson) [reply CONTRIBUTING.md]: I disagree with this suggestion. I prefer the original wording.

**Pull request merged**